app.js

```diff
@@ -1,13 +1,14 @@
 const express = require('express')
-const { getCategories } = require('./controllers/controller')
+const { getCategories } = require('./controllers/categories_controller')
+const { getReveiws } = require('./controllers/reviews_controller')
```

Comment:
Typo on getReviews here

Reply:
Done, amusingly it still worked because I made the same typo in the controller lol